docs/api/components/schemas.yaml

```diff
@@ -2026,7 +2026,7 @@ Startup-config:
             By default, this will only be accessible to the localhost.
           type: string
           default: '127.0.0.1:4985'
-        metric_interface:
+        metrics_interface:
```

Comment:
This should get removed, since we don't document unsupported options.

Reply:
This was a typo i fixed since the metrics_interface_authentication option also says metrics, should i remove it?

Reply:
Understood.